src/applications/personalization/dashboard/components/FormItem.jsx

```diff
@@ -12,80 +12,43 @@ import DashboardAlert, {
   DASHBOARD_ALERT_TYPES,
 } from 'applications/personalization/dashboard/components/DashboardAlert';
 
-class FormItem extends React.Component {
-  render() {
-    const savedFormData = this.props.savedFormData;
-    const formId = savedFormData.form;
-    const {
-      lastUpdated: lastSaved,
-      expiresAt: expirationTime,
-    } = savedFormData.metadata;
-    const lastSavedDateTime = moment
-      .unix(lastSaved)
-      .format('MMMM D, YYYY [at] h:mm a');
-    const expirationDate = moment.unix(expirationTime).format('MMMM D, YYYY');
-    const isExpired = moment.unix(expirationTime).isBefore();
-    const itemTitle = `Application for ${formTitles[formId]}`;
-    const formIdTitle = presentableFormIDs[formId];
+function FormItem({ savedFormData }) {
```

Comment:
sorry, ESLint made me refactor this to a functional component. The main change is that this only shows a non-expired view. I removed all code related to the expired view.